warehouse/templates/manage/project/history.html

```diff
@@ -41,7 +41,7 @@ <h2>{% trans %}Security history{% endtrans %}</h2>
     </strong>
     <small>
       {% trans %}Added by:{% endtrans %}
-      {% if event.additional.publisher_url %}
+      {% if event.additional.uploaded_via_trusted_publisher or event.additional.publisher_url %}
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought (non-blocking):** ‏Seems like we ought to have one or the other, but not both. Probably only `uploaded_via_trusted_publisher` and do a data migration?

Reply:
A minor hiccup here is that we want to render `publisher_url` in some scenarios (e.g., for GitHub, it'll point to the specific commit of the workflow that was run). So I think we need to preserve both JSON keys, unfortunately.

(Maybe there's a better way to do this, though?)

Reply:
Yep, that summary is correct! We effectively have one old variable (`publisher_url`) that we also want to render, one new variable (`uploaded_via_trusted_publisher`) which is just a bool for checking, and the two only partially overlap 🙃 

A data migration for the older events makes sense to me. I can work on that in a follow-up.

Reply:
(Or alternatively, the column addition would fix all of this.)